### DISCUSSION TITLE
I am having more two pattern validation in the same fields. How to implement that ?

### DISCUSSION THREAD

**Discussion by Sharathvasudevan09051991** [react-hook-form/react-hook-form#2497] [Q&A]
I am having more two pattern validation in the same fields. How to implement that ?

**Comment by bluebill1049** [comment]
`validate: { method1, method2 }`

**Reply by Sharathvasudevan09051991** [reply]
Any Link for the implementation.

**Reply by Sharathvasudevan09051991** [reply]
Getting error like this : unexpected token , expected ","

**Reply by Sharathvasudevan09051991** [reply]
{errors.fieldname && errors.score.type === "validate.method1" && (
<div class="error">Method one error</div>
)
{errors.fieldname && errors.score.type === "validate.method2" && (
<div class="error">Method two error</div>
)

Is this right or how to display two different error message

**Reply by bluebill1049** [reply]
https://react-hook-form.com/api#register
<img width="985" alt="Screen Shot 2020-08-07 at 8 24 38 am" src="https://user-images.githubusercontent.com/10513364/89588660-70a0ea80-d887-11ea-9495-c770ea0ccb10.png">
